horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/RunService.java

```diff
@@ -222,6 +222,7 @@ Response add(@QueryParam("test") String testNameOrId,
             @Parameter(name = "description", description = "Run description", example = "AWS runs"),
 
     })
+    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
I had to explictly state the api produces `text/plain` to fix this issue in the UI:
![image](https://github.com/user-attachments/assets/2e29d4ce-c778-44c1-85aa-23c153567c8f)

Reply:
atm it just returns an array of ID's for the accepted Runs to process.
we probably should return a properly formatted JSON object, that contains the URI for each run, e.g https://restfulapi.net/http-status-202-accepted/

For now, returning as plain text is OK

Reply:
yeah I agree, I think this is part of a wider refactoring of the API as we have other endpoints that are not really compliant with the REST standards